### DISCUSSION TITLE
Transfer of Libre 3 sensor to another phone using Juggluco?

### DISCUSSION THREAD

**Discussion by dar-kod** [maheini/FreeStyle-Libre-3-patch#60] [Q&A]
I am currently using the FreeStyle Libre 3 sensor with the Juggluco app on a phone where the sensor was initially activated without a LibreView account. This was done with the penultimate version of **Juggluco 8.2.9**. I would like to know if it is possible to transfer the active sensor connection to a new phone that also uses the Juggluco app, either through the "Mirror", "Sync" or "Reinit" options, or any other method.

For example:

Can the same sensor be used on the new phone without losing functionality?

If not, are there specific limitations in Juggluco that prevent this, and do you plan to enable this functionality in future updates?

Also the "Settings" screen looks different in the latest version of **Juggluco 9.0.4**. There is no option for xDrip Broadcast and Patched Libre. It is not clear how Juggluco is connected to xDrip.

**Comment by maheini** [comment]
Hi @dar-kod 
It's totally possible if you take into account some important steps:

Prerequisites (Those seem to be good in your case)
1. Login to Librelink account before activating the new sensor.
2. Now activate the sensor within the same app you just logged in. 

Your sensor should now be linked to your librelink account. You should now be able to switch phones. In order to prevent any issues, here's a quick instruction:

1. Turn off Bluetooth on your active phone, in order to force the sensor to disconnect. Keep Bluetooth off. Don't delete Juggluco on the old phone yet - let's keep it as a backup :)
2. Log in to the Juggluco app on your new phone, using the **same** librelink account of your previous phone.
4. After logging in, you should be good to start the sensor again on your new phone. After some minutes, there should be new readings.
5. As soon as there are new readings, you can safely enable Bluetooth back on on your old phone.

If there are problems:
- Keep Bluetooth disabled on the old phone
- Wait for 60 minutes on your phone
- Verify if the login was successful in Juggluco (You should see a librelink id within the settings)
- Try to re-start the sensor again.
- Uninstall Libre 3 app on your new phone, in order to prevent interferences.
- If it still doesn't work, try to re-start the sensor using the libre 3 app - are there any error messages?

**Comment by dar-kod** [comment]
Thank you, but the "Prerequisites" are not clear. Do I need to scan the sensor a second time with the Juggluco on the old phone? When I go to "Settings" -> Libreview, it seems that it is already connected because there is a number between "Change start" and "Get Account ID".

Then you wrote:

>2. Log in to the Juggluco app on your new phone, using the same librelink account of your previous phone.

Here is a problem: in the Juggluco 9.0.4 application on the new phone, there is no option for Libreview in the "Settings".

**Reply by maheini** [reply]
If you weren't logged in with librelink at the point you activated the sensor for it's first time, theres no way to "fix" it later on. But as far as I know, you already did that, right?

In the newest Juggluco version, you can find Libreview within Settings -> Data broadcast -> Libreview.

**Reply by dar-kod** [reply]
I have just logged into Libreview.com to see if my current sensor data is there. Yes, it is.

**Reply by maheini** [reply]
Yey - looks like you're good to go then 😃

**Reply by maheini** [reply]
There's nothing to lose - in the worst case you can just switch back to the old phone... by enabling BT back on.

**Reply by dar-kod** [reply]
Thank you, the transfer happened immediately after turning off Bluetooth and scanning a sensor. When everything is ready to transfer, including AAPS, xDrip+ and Juggluco, it takes a few minutes for everything to transfer and work on another phone.
